grails-app/migrations/changelog.groovy

```diff
@@ -1,9 +1,75 @@
+import util.LiquibaseUtil
+import util.TaggedMigrationVersion
+
+/*
+ * Because we skip changelogs under the release folders (Ex: '0.9.x') when doing clean installs, we don't get tagged
+ * changesets, and so we need to manually define what release the 'install' version of the migration brings us to. We
+ * don't want to assume that it always brings us to the latest release because we may have forgotten or decided not to
+ * update/regenerate the 'install' migrations during the release process.
+ */
+final TaggedMigrationVersion CLEAN_INSTALL_VERSION = new TaggedMigrationVersion('0.8.x')
+
+/*
+ * List all of our tagged migration versions. Any new tagged versions should be added to this list.
+ * By convention, these should map to the migrations directories.
+ */
+final List<TaggedMigrationVersion> ALL_VERSIONS = [
```

Comment:
I thought about having it read all the folder names in the /migrations folder to extract these versions so that we wouldn't need to update this list with every release but that felt pretty hacky so I left it like this. We're already expecting that specific folder structure though so maybe it's fine to do.